src/bvals/comms/boundary_communication.cpp

```diff
@@ -314,8 +314,8 @@ TaskStatus ApplyCoarseBoundaryConditions(std::shared_ptr<MeshData<Real>> &md) {
 TaskID AddBoundaryExchangeTasks(TaskID dependency, TaskList &tl,
                                 std::shared_ptr<MeshData<Real>> &md, bool multilevel) {
   const auto any = BoundaryType::any;
-  const auto local = BoundaryType::local;
-  const auto nonlocal = BoundaryType::nonlocal;
+  // const auto local = BoundaryType::local;
+  // const auto nonlocal = BoundaryType::nonlocal;
```

Comment:
Is there a reason to keep the commented-out code here?

Reply:
The local vs nonlocal comms split is a useful thing to keep in mind. There's more commented out code below those lines. Presumably @pgrete added those lines in particular for a warning.

Reply:
It might be useful to have an inline comment that explains why this is commented-out.

Reply:
I think @lroberts36 made some comment about this in another part of the code---it has to do with sparse comms.

Reply:
The comment is actually a couple of lines down at the end of the commented code.
Those two lines were the first set of lines that are related to a block of code splitting communication between local and nonlocal blocks (which is currently defunct).
So we want to keep this code (but commented so that there's no compiler warning).